tasks/db_deploy/test/unit_tests/test_migrate_db.py

```diff
@@ -53,93 +57,22 @@ def test_perform_migration_happy_path(
 
                 if version == 1:
                     mock_migrate_v1_to_v2.assert_called_once_with(self.config, False)
-                    mock_migrate_v2_to_v3.assert_called_once_with(self.config, True)
+
                 elif version == 2:
-                    mock_migrate_v2_to_v3.assert_called_once_with(self.config, True)
+                    mock_migrate_v2_to_v3.assert_called_once_with(self.config, False)
```

Comment:
If version is 2 shouldn't all but the v1_v2 migration be called?

Reply:
added